.snippets/text/node-operators/validators/onboarding/onboarding-timeline.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+      "title": "[Run an operator node](/node-operators/validators/onboarding/run-a-validator/){target=_blank}",
+      "content": "Set up the actual node using Systemd or Docker.",
+      "icon": ":fontawesome-solid-1:"
+    },
+    {
+      "title": "[Register in Symbiotic](/node-operators/validators/onboarding/register-in-symbiotic/){target=_blank}",
+      "content": "Add your account to the Symbiotic operators registry.",
```

Comment:
Why do they have to do this? A very brief explanation would be helpful at this point